### PR TITLE
Style dgraph nodes in Graph Overview

### PR DESCRIPTION
**Pull request by mdroidian**
https://www.loom.com/share/6c14c60fd1f6462ca71fd117f04734d0

### CODE REVIEW

File: src/components/TldrawCanvas.tsx [hunk 1]
```diff
@@ -234,10 +235,15 @@ const DEFAULT_WIDTH = 160;
...
+// FONT_FAMILIES.sans or tldraw_sans not working in toSvg()
+// maybe check getSvg()
+// in node_modules\@tldraw\tldraw\node_modules\@tldraw\editor\dist\cjs\lib\app\App.js
+const SVG_FONT_FAMILY = "sans-serif";
+
 export const DEFAULT_STYLE_PROPS = {
   ...TEXT_PROPS,
   fontSize: FONT_SIZES.m,
-  fontFamily: "sans",
+  fontFamily: FONT_FAMILIES.sans,
```
Comment (mdroidian) [inline src/components/TldrawCanvas.tsx:246]: Unrelated fix

File: src/components/TldrawCanvasLabelDialog.tsx [hunk 2]
```diff
@@ -124,20 +125,7 @@ const LabelDialog = ({
...
-      (s): s is QBClause =>
-        s.type === "clause" && s.relation === "has title" && s.source === text
-    );
-    if (!titleCondition) return "";
-    return titleCondition.target
-      .replace(/^\/(\^)?/, "")
-      .replace(/(\$)?\/$/, "")
-      .replace(/\\\[/g, "[")
-      .replace(/\\\]/g, "]")
-      .replace(/\(\.[\*\+](\?)?\)/g, "");
```
Comment (mdroidian) [inline src/components/TldrawCanvasLabelDialog.tsx:140]: Extracted to function in `discourseGraphsMode.ts`

File: src/discourseGraphsMode.ts [hunk 3]
```diff
@@ -760,6 +792,61 @@ const initializeDiscourseGraphsMode = async (args: OnloadArgs) => {
...
+        getSetting: (settingName: string) => any;
+      };
+      type nodeData = {
+        x: number;
+        y: number;
+        label: string;
+        size: number;
+      };
+
+      window.roamAlphaAPI.ui.graphView.wholeGraph.addCallback({
```
Comment (dvargas92495) [inline src/discourseGraphsMode.ts:806]:
```typescript
unloads.add(function removeGraphViewCallback() {
   label: "discourse-node-styling",
})
```

**Review by dvargas92495 — APPROVED** [review]
pending unload func
